Tools/test-updater.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+from Crypto.PublicKey import ECC
```

Comment:
Could you have a toplevel comment in this file with the non-standard dependencies? Looks like it's only Cryptodome right now?